libs/langchain-mistralai/src/chat_models.ts

```diff
@@ -235,7 +210,7 @@ function convertMessagesToMistralMessages(
       content,
       tool_calls: toolCalls,
     };
-  });
+  }) as MistralAIMessage[];
```

Comment:
Their sdk types don't play well with their API... casting allows us to use this type which doesn't throw errors from their API